api/api/routers/mcp/_mcp_models.py

```diff
@@ -16,6 +17,55 @@
 from core.utils.fields import datetime_zero
 
 
+class ConciseLatestModelResponse(BaseModel):
+    id: str
+    currently_points_to: str
+
+
+class ConciseModelResponse(BaseModel):
+    id: str
+    maker: str
+    display_name: str
+    supports: list[str]
+    quality_index: int
+    cost_per_input_token_usd: float
+    cost_per_output_token_usd: float
+    release_date: str
+
+    @classmethod
+    def from_model_data(cls, id: str, model: FinalModelData):
+        IGNORE_SUPPORTS = {"structured_output", "support_system_messages", "json_mode"}
+
+        provider_data = model.providers[0][1]
+        return cls(
+            id=id,
+            maker=model.provider_name,
+            display_name=model.display_name,
+            supports=[
```

Comment:
is that going to look weird when serialized in JSON ? we should maybe serialize to strings ?

Reply:
@guillaq that's just a list of strings:


 ```
 "supports": [
          "input_image",
          "output_text",
          "json_mode",
          "structured_output",
          "parallel_tool_calls",
          "tool_calling"
        ],
```